docker-compose.yml

```diff
@@ -90,3 +88,7 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
+
+networks:
```

Comment:
@AgustinRamiroDiaz should we have this reference to feed_buzz here? Can we maybe find a more generic name?

Reply:
My bad, this shouldn't have made it into the PR. I'll remove it